oiio-sys/build.rs

```diff
@@ -17,12 +17,21 @@ fn main() -> Result<()> {
         return Ok(());
     }
 
-    let pkgconfig = pkg_config::probe_library("OpenImageIO")?;
+    let pkgconfig_ooio = pkg_config::probe_library("OpenImageIO")?;
```

Comment:
This is me being picky, but `pkgconfig_oiio`.

Reply:
This is not being picky. I'm embarrassed and will fix it :)